collectors/template/cfn/collector.template.template

```diff
@@ -0,0 +1,102 @@
+{
```

Comment:
If this file is being moved, the path in the template script needs to change as well.

https://github.com/alertlogic/paws-collector/blob/master/create-collector.js#L41